scripts/vectors/python/cbor_canon.py

```
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+import sys, json
+try:
+    import cbor2  # pip install cbor2
+except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)